.test_patterns.yml

```diff
@@ -138,6 +138,10 @@ tests:
     error_regex: "ContractFunctionExecutionError.*execution failed"
     owners:
       - *palla
+  - regex: "src/e2e_p2p/preferred_gossip_network.test.ts"
+    error_regex: "muxer closed locally"
+    owners:
+      - *palla
```

Comment:
<img width="897" height="500" alt="image" src="https://github.com/user-attachments/assets/b7536622-6b04-48a8-b482-5b5f9c49e31d" />

Reply:
Happy to be the scapegoat here!